ReactNativeClient/lib/services/KeymapService.ts

```diff
@@ -0,0 +1,202 @@
+		{ accelerator: 'CommandOrControl+Q', command: 'quit', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+,', command: 'preferences', platforms: ['darwin'] },
+		{ accelerator: 'Command+W', command: 'closeWindow', platforms: ['darwin'] },
+		{ accelerator: 'CommandOrControl+Alt+I', command: 'insertTemplate', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+C', command: 'copy', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+X', command: 'cut', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+V', command: 'paste', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+A', command: 'selectAll', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+B', command: 'textBold', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+I', command: 'textItalic', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+K', command: 'textLink', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+`', command: 'textCode', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+T', command: 'insertDateTime', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: shim.isMac() ? 'Shift+Command+F' : 'F6', command: 'focusSearch', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+F', command: 'showLocalSearch', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+S', command: 'focusSidebar', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+L', command: 'focusNoteList', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+N', command: 'focusNoteTitle', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+B', command: 'focusNoteBody', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: shim.isMac() ? 'Option+Command+S' : 'F10', command: 'toggleSidebar', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+L', command: 'toggleVisiblePanes', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+0', command: 'zoomActualSize', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+E', command: 'startExternalEditing', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Alt+T', command: 'setTags', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+,', command: 'options', platforms: ['linux', 'win32', 'freebsd'] },
```

Comment:
You have twice the same shortcut - "options" and "preferences". The preferences are generally called "config screen" in the doc, so let's called the shortcut "config".